src/test/java/com/google/cloud/tools/managedcloudsdk/install/DownloaderTest.java

```diff
@@ -101,8 +101,7 @@ public void testDownload() throws IOException, InterruptedException {
 
     ProgressVerifier.verifyProgress(
         mockProgressListener,
-        String.format(
-            Locale.getDefault(), "Downloading %.2f MB", testFileSize / 1024.0f / 1024.0f));
+        String.format(Locale.getDefault(), "Downloading %.2f MB", 0.078125954f));
```

Comment:
again, this can go all the way to a single string literal. I don't think you need String.format here.

Reply:
I mostly did this to keep the locale specification in, should I just remove it from the test and assume US?

Reply:
You don't have to assume the locale. You can specify it at the test suite level. I.e. in .travis.yaml